api/settings.py

```diff
@@ -40,6 +40,13 @@ def assemble_db_connection(cls, v: Optional[str], values: Dict[str, Any]) -> Any
     EMAIL_RESET_TOKEN_EXPIRE_HOURS: int = 8
     ALGORITHM: str = "HS256"
 
+    # CORS
+
+    CORS_ORIGINS: List[AnyHttpUrl] = ["http://www.example.com"]  # type: ignore
```

Comment:
Opened an issue on pydantic regarding mypy checks of AnyHttpUrl. 

https://github.com/samuelcolvin/pydantic/issues/3143